requirements.txt

```diff
@@ -17,4 +18,4 @@ pytz==2018.4
 scipy==1.0.1
 setuptools==39.0.1
 six==1.11.0
-coloredlogs==10.0
+sklearn==0.0
```

Comment:
sklearn version 0 ?

Reply:
I know, that's from my pip freeze (after an upgrade).

Reply:
Should I trust pip freeze ? I don't know what version to write

Reply:
From a brief google search, I found this:
https://stackoverflow.com/questions/46991795/why-does-pip-freeze-geneate-package-name-0-0-0

which, according to the question, makes the package installation break. I would guess it is safer to remove this line

Reply:
After 30 seconds more of research, I came across this:
https://github.com/scikit-learn/scikit-learn/issues/8215
This does provide more information, but no answers

Reply:
Actually it was a wrong commit from another PR. I removed it.
But I will use scikit-learn==0.22.1 now (I think it is an alias problem)